### PR TITLE
feature/Connect firebase

### PR DESCRIPTION
**Pull request by Dominic-go**
## Ticket

- [Firebase](https://trello.com/c/7sNvexcV/1-setup-project)

## Task

- [x] Firebase console and connect

### CODE REVIEW

File: lib/main.dart [hunk 1]
```diff
@@ -1,8 +1,21 @@
...
+  await Firebase.initializeApp(
+    options: DefaultFirebaseOptions.currentPlatform,
   );
+  
+  await dot_env.dotenv.load(
+    fileName: "assets/.env",
+  );
+  FlutterError.onError = FirebaseCrashlytics.instance.recordFlutterFatalError;
+  FirebaseCrashlytics.instance.crash();
+
```
Comment (drake-ho-goldenowl) [inline lib/main.dart]: What is the mean of the line 18? 
And reference boilerplate about calling Firebase Crashlytics in main function

Comment (Dominic-go) [reply lib/main.dart]: Line 18 mean create a crash intentionally to save the necessary information when testing.

To use Firebase Crashlytics, I used FlutterError.onError = FirebaseCrashlytics.instance.recordFlutterFatalError; to link my Flutter app's error handling to Firebase Crashlytic.

Comment (drake-ho-goldenowl) [reply lib/main.dart]: line 18 is used for testing but you still committed

Comment (drake-ho-goldenowl) [reply lib/main.dart]: follow the boilerplate to call Firebase Crashlytics

**Review by drake-ho-goldenowl — APPROVED** [review]

**Pull request merged**